pkg/cmd/cli/cli.go

```diff
@@ -72,6 +74,21 @@ func NewCommandCLI(name string) *cobra.Command {
 	return cmds
 }
 
+// applyToCreate injects the deprecation notice about for 'apply' command into
+// 'create' command.
+// TODO: Remove this once we get rid of 'apply' in all documentation/etc.
+func applyToCreate(dst *cobra.Command) *cobra.Command {
```

Comment:
How about creating a generic deprecation function that we'll be able to use later?